docs/api/routing.rst

```diff
@@ -374,6 +374,60 @@ A custom router is any class that implements the following interface:
 
             """
 
+Suffixed Responders
+-------------------
+
+While Falcon encourages the REST architectural style, it is flexible enough to accomodate other architectures. Consider
```

Comment:
Maybe it's just me, but IMHO "other architectures" is a bit imprecise; i.e. an _architecture_ is not really good synonym for an _architectural style_.

If we want to avoid the "architecture style" in the same sentence because of repetitiveness, maybe "other API styles", "other paradigms" or simply "other styles" could do?

Reply:
I see what you're saying, let me replace with "other paradigms" then. I think that sounds good.

Reply:
Updated